app.py

```diff
@@ -167,9 +167,9 @@
     container_port=4200,
     container_memory=200,
     container_env_vars={
-        "API_DOCS_URL": f"http://{fully_qualified_domain_name}/api-docs",
+        "API_DOCS_URL": "http://agora-api-docs/api-docs",
         "APP_VERSION": f"{agora_version}",
-        "CSR_API_URL": f"http://{fully_qualified_domain_name}/api/v1",
+        "CSR_API_URL": "http://agora-api:3333/api/v1",
```

Comment:
I'm only speculating that this will work because it makes sense.  However I think @tschaffter would know for sure since he came up with this setup.

Reply:
I think you're right.  We will test this after a merge.

Reply:
Will removing break the agora app?  Does it need a `CSR_API_URL` env var to be defined?

Reply:
We need this as this is for client side rendering of the app